src/stp/plays/offensive/AttackingPass.cpp

```diff
@@ -33,14 +29,16 @@ namespace rtt::ai::stp::play {
     }
 
 AttackingPass::AttackingPass() : Play() {
-    startPlayInvariants.clear();
-    startPlayInvariants.emplace_back(std::make_unique<invariant::NormalPlayGameStateInvariant>());
-    startPlayInvariants.emplace_back(std::make_unique<invariant::BallCloseToUsInvariant>());
-    startPlayInvariants.emplace_back(std::make_unique<invariant::NoGoalVisionFromBallInvariant>());
-    startPlayInvariants.emplace_back(std::make_unique<invariant::BallClosestToUsInvariant>());
+    startPlayEvaluation.clear();
+    startPlayEvaluation.emplace_back(GlobalEvaluation::NormalPlayGameState);
+//    startPlayEvaluation.emplace_back(GlobalEvaluation::BallCloseToUs);
+//    startPlayEvaluation.emplace_back(GlobalEvaluation::NoGoalVisionFromBall);
+//    startPlayEvaluation.emplace_back(GlobalEvaluation::BallClosestToUs);
```

Comment:
Why is this commented? atleast provide a comment so people (like you) know in the future